.github/workflows/wheels.yml

```diff
@@ -112,63 +120,47 @@ jobs:
         if: ${{ runner.os == 'Windows' && env.IS_32_BIT == 'false' }}
 
       - name: Setup macOS
-        if: matrix.buildplat[0] == 'macos-11' || matrix.buildplat[0] == 'macos-14'
+        if: matrix.buildplat[0] == 'macos-13' || matrix.buildplat[0] == 'macos-14'
         run: |
+          # Symlink gfortran installed on runner
+          GFORTRAN_LOC=$(which gfortran-13)
+          ln -s $GFORTRAN_LOC gfortran
+          export PATH=$PWD:$PATH
+          echo "PATH=$PATH" >> "$GITHUB_ENV"
+          GFORTRAN_LIB=$(dirname `gfortran --print-file-name libgfortran.dylib`)
+          MACOSX_TARGET=${{ matrix.buildplat[4] }}
+          if [[ ${{ matrix.buildplat[3] }} == 'accelerate' ]]; then
+            echo CIBW_CONFIG_SETTINGS=\"setup-args=-Dblas=accelerate\" >> "$GITHUB_ENV"
+          fi
           if [[ ${{ matrix.buildplat[2] }} == 'arm64' ]]; then
-            # macosx_arm64
```

Comment:
the way that the diff is generated makes it hard to review.

Reply:
Sorry about that. I did some refactoring which meant changing indentation levels for many lines. Overall, I do think this approach is much cleaner and avoid duplicated code.